Source/ToolCore/JSBind/JSBClass.h

```diff
@@ -136,6 +136,9 @@ class JSBClass : public Object
     JSBModule* GetModule() { return module_; }
     JSBPackage* GetPackage() { return module_->GetPackage(); }
 
+    bool HasPrivateConstruct() { return hasPrivateConstruct_; }
+    void SetHasPrivateConstruct(bool value = true) { hasPrivateConstruct_ = value; }
+
```

Comment:
I am not seeing this API addition used anywhere, also a bit questionable, skipping the constructor in the header traversal will mean there isn't a constructor function in the class, which can be tested without needing a local member variable